hoodie-common/src/main/java/com/uber/hoodie/common/io/storage/HoodieWrapperFileSystem.java

```diff
@@ -377,7 +399,15 @@ public FSDataOutputStream createNonRecursive(Path f, FsPermission permission,
 
   @Override
   public boolean createNewFile(Path f) throws IOException {
-    return fileSystem.createNewFile(convertToDefaultPath(f));
+    boolean newFile = fileSystem.createNewFile(convertToDefaultPath(f));
```

Comment:
should we cover `mkdirs` as well? also createNonRecursive and other write methods? Can you skim this file for any write operation that we may have to consider?

Reply:
Fixed